frontend/packages/kubevirt-plugin/console-extensions.json

```diff
@@ -1,4 +1,15 @@
 [
+  {
+    "type": "console.flag/model",
+    "properties": {
+      "model": {
+        "group": "kubevirt.io",
+        "version": "v1alpha3",
+        "kind": "VirtualMachine"
+      },
+      "flag": "KUBEVIRT"
```

Comment:
Note that in general, having the same feature flag determined by different models might lead to race condition, but in this particular case it's OK because we don't expect two different `VirtualMachine` CRDs (`v1` alongside `v1alpha3`) to be present on the cluster.